com.unity.ml-agents/Runtime/DemonstrationStore.cs

```diff
@@ -5,11 +5,14 @@
 namespace MLAgents
 {
     /// <summary>
-    /// Responsible for writing demonstration data to stream (usually a file stream).
+    /// Responsible for writing demonstration data to stream (typically a file stream).
     /// </summary>
     public class DemonstrationStore
     {
-        public const int MetaDataBytes = 32; // Number of bytes allocated to metadata in demo file.
+        /// <summary>
+        /// Number of bytes allocated to metadata in a demo file.
```

Comment:
Maybe a better description:
```suggestion
        /// Number of bytes reserved for the Demonstration metadata at the start of the demo file.
```